### DISCUSSION TITLE
Improved Heavy Armor, Craftable Crafting Armor spreadsheet

### DISCUSSION THREAD

**Discussion by Valo56** [zunath/SWLOR_NWN#1507] [General]
This has two things.

The first, I'm not sure if it's desirable. I am of the belief though that heavy armor is the weakest option. It does not provide adequate sustain to players. Light and Force armor provides stamina and force points, respectively, which provide enormous sustain through healing. With heavy armor you get a bit more defense and a larger HP pool, which is good, but without lots of stamina and force points that HP pool is all the harder to maintain.

Given Heavy Armor requires its own perk as well, it's not possible to mix and match without SIGNIFICANT perk point investment. This leads to heavily armored PC's basically stuck going pure heavy armor where they have an HP pool well beyond their means to maintain.

To address this I have two .erf's here. The first one doesn't include HP regen, the second does. I believe including both of them would solve heavy armor to be viable without leaving it too strong. **Basically, the first one provides all craftable T3-T5 heavy armor with elemental resistances equal to half their force defense, rounded down.**

The second one provides all chest and shield slot items with regeneration as well, equal to their Tier * 2. This provides heavy armor users access to up to 20 HP Regen, the equivalent of always having a Kolto Grenade up, if they focus on that.

The addition of elemental resists also provides heavy armor added functionality as a tank.

Basically the whole thing is designed to help tanks be tanks. They aren't there to be flashy and crush everything, they're there to get hit and keep standing throughout it so others can do their job.

Anyway, other than that, I also have this here. Unfortunately I haven't been able to make the items for the new Enhancements, but here's the new recipes for whenever you have a moment. The second tab has all the crafting equipment recipes. I kept them as having very high level for their tier with significant resource requirements. The enhancements on the first tab should not be added yet as it is not possible to create items with those properties without hak updates.

https://docs.google.com/spreadsheets/d/1MmMJAbs9-6rr43_XQiG3dmCBRZrkEQqrZnwgeOUFd6Q/edit?usp=sharing

For both .erf's below, the following are the affected resrefs. Only one file is needed.
![image](https://user-images.githubusercontent.com/106187784/180935576-5fd118dc-f4fc-4a73-9ec9-014f43722e65.png)

Heavy Armor (No Regen)
https://www.dropbox.com/s/crzaohfu8znaxp7/heavyarmor.erf?dl=0
Heavy Armor (Regen)
https://www.dropbox.com/s/px2c4s4oql1byb5/heavyarmorregen.erf?dl=0

Obviously, I don't necessarily expect this to be added. It was an idea I had and am throwing it out there for easy inclusion.

**Comment by zunath** [comment]
We went a different way with Wild's rebalance efforts. I'll convert this to a discussion.
